components/raftstore/src/store/config.rs

```diff
@@ -145,14 +148,13 @@ pub struct Config {
     pub local_read_batch_size: u64,
 
     #[config(skip)]
-    pub apply_max_batch_size: usize,
-    #[config(skip)]
-    pub apply_pool_size: usize,
+    #[serde(flatten, with = "prefix_apply")]
+    pub apply_batch_system: BatchSystemConfig,
 
     #[config(skip)]
-    pub store_max_batch_size: usize,
-    #[config(skip)]
-    pub store_pool_size: usize,
+    #[serde(flatten, with = "prefix_store")]
```

Comment:
How about `#[serde(rename = "...")]`?

Reply:
`#[serde(rename = "...")]` does not work.